models/ignore_and_fire.h

```diff
@@ -0,0 +1,325 @@
+/*
+ *  ignore_and_fire.h
+ *
+ *  This file is part of NEST.
+ *
+ *  Copyright (C) 2004 The NEST Initiative
+ *
+ *  NEST is free software: you can redistribute it and/or modify
+ *  it under the terms of the GNU General Public License as published by
+ *  the Free Software Foundation, either version 2 of the License, or
+ *  (at your option) any later version.
+ *
+ *  NEST is distributed in the hope that it will be useful,
+ *  but WITHOUT ANY WARRANTY; without even the implied warranty of
+ *  MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+ *  GNU General Public License for more details.
+ *
+ *  You should have received a copy of the GNU General Public License
+ *  along with NEST.  If not, see <http://www.gnu.org/licenses/>.
+ *
+ */
+
+#ifndef IGNORE_AND_FIRE_H
+#define IGNORE_AND_FIRE_H
+
+// Includes from nestkernel:
+#include "archiving_node.h"
+#include "connection.h"
+#include "event.h"
+#include "nest_types.h"
+#include "ring_buffer.h"
+#include "universal_data_logger.h"
+
+namespace nest
+{
+
+/* BeginUserDocs: neuron, integrate-and-fire, current-based
+
+Short description
++++++++++++++++++
+
+Neuron generating spikes at fixed intervals irrespective of inputs.
+
+Description
++++++++++++
+
+The ``ignore_and_fire`` neuron is a neuron model generating spikes with a fixed predefined inter-spike interval,
+irrespective of its inputs ("ignore"). In this simplest version of the ignore_and_fire neuron, the inputs from other
+neurons or devices are not processed at all (*). The ``ignore_and_fire`` neuron is primarily used for neuronal-network
+model validation and benchmarking purposes, in particular to evaluate the correctness and performance of connectivity
+generation and inter-neuron communication. It permits an easy scaling of the network size and/or connectivity without
+affecting the output spike statistics. The amount of network traffic is predefined by the user, and therefore fully
+controllable and predictable, irrespective of the network size and structure.
+
```

Comment:
Here, I would suggest to add the remaining explanatory text from the ACA testcases:

"To generate asynchronous activity for a population of ``ignore_and_fire`` neurons, the firing ``phase``s can be randomly initialized. Note that the firing ``phase`` is a real number, defined as the time to the next spike relative to the firing period."